bitsail-test/bitsail-test-end-to-end/bitsail-test-e2e-base/src/test/java/com/bytedance/bitsail/test/e2e/executor/generic/GenericExecutorSettingTest.java

```diff
@@ -46,9 +46,9 @@ public void testReadSetting() throws Exception {
     Assert.assertTrue(transferableFileList.contains(file1));
     Assert.assertTrue(transferableFileList.contains(file2));
 
-    Assert.assertEquals("pwd && sleep 5000",
+    Assert.assertEquals("pwd && sleep 1",
```

Comment:
And why reduce this sleep time?

Reply:
Just for reduce waiting time for testing.